SDK/MobileCenterCrashes/Microsoft.Azure.Mobile.Crashes.Shared/Crashes.cs

```diff
@@ -1,37 +1,29 @@
 ﻿using System;
+using System.Diagnostics;
 using System.Threading.Tasks;
 
 namespace Microsoft.Azure.Mobile.Crashes
 {
     /// <summary>
     /// Crashes service.
     /// </summary>
-    public static class Crashes
+    public partial class Crashes
```

Comment:
Analytics and crashes need to be static so that they can inherit from IMobileCenterService. Added the constructors though